git/squash.md

```diff
@@ -0,0 +1,16 @@
+# How and Why to Squash Merge your Pull Requests
+
+[![How and Why to Squash Merge your Pull Requests](http://i3.ytimg.com/vi/41EITMtWUZM/maxresdefault.jpg)](https://www.youtube.com/watch?v=41EITMtWUZM "How and Why to Squash Merge your Pull Requests")
```

Comment:
@spaceninja , heads up: this video is private so I can't watch it.

Reply:
Thanks! Changed to unlisted, so now it should work